docs/09.EXT-REFERENCE-CONTEXT.md

```diff
@@ -0,0 +1,142 @@
+# jerryx_context types
```

Comment:
I would prefer a separate file for each extension. And I wouldn't mind removing this numbering, since they show creation order. I am not sure it is useful.